packages/gatsby-cli/src/reporter/reporters/ink/components/develop.js

```diff
@@ -44,7 +44,7 @@ class Develop extends Component {
     })
   }
 
-  componentWillUpdate(nextProps) {
+  UNSAFE_componentWillUpdate(nextProps) {
```

Comment:
I think we should defer this change to after we move to React 17 

From the documentation at https://reactjs.org/docs/react-component.html

> That name will continue to work until version 17